tests/rustdoc-ui/crate-reference-in-block-module.rs

```diff
@@ -0,0 +1,5 @@
+// check-pass
```

Comment:
Empty `.stderr` tests should not be necessary for `// check-pass` tests.

Reply:
The test shows as ignored for me without it.